packages/widgets/src/rssFeed/component.tsx

```diff
@@ -3,15 +3,29 @@ import { Card, Flex, Group, Image, ScrollArea, Stack, Text } from "@mantine/core
 import { IconClock } from "@tabler/icons-react";
 import dayjs from "dayjs";
 
+import { clientApi } from "@homarr/api/client";
+
 import type { WidgetComponentProps } from "../definition";
 import classes from "./component.module.scss";
 
-export default function RssFeed({ serverData, options }: WidgetComponentProps<"rssFeed">) {
-  if (serverData?.initialData === undefined) {
-    return null;
-  }
+export default function RssFeed({ options, itemId }: WidgetComponentProps<"rssFeed">) {
+  const [rssFeeds] = clientApi.widget.rssFeed.getFeeds.useSuspenseQuery(
+    {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      itemId: itemId!,
```

Comment:
Can we always expect this to be not null?

Reply:
Always except when you are on the try out page. IMO it would make generally more sense to just send the options to the server